Source/DataSource.swift

```diff
@@ -42,29 +42,35 @@ public struct DataSource<Item: ParentType> {
     public mutating func insert(item: Item, at indexPath: IndexPath) {
         insert(item: item, atRow: indexPath.row, inSection: indexPath.section)
     }
-    
+
     /// Inserts the item at the specified row and section.
     ///
     /// - Parameters:
     ///   - item: The item to insert.
     ///   - row:  The row index of the item.
     ///   - section: The section index of the item.
     public mutating func insert(item: Item, atRow row: Int, inSection section: Int) {
-        guard section < numberOfSections() else { return }
-        guard row <= numberOfItems(inSection: section) else { return }
+        guard section < numberOfSections() else {
```

Comment:
It would be very helpful to keep the current style guide of the repository, it makes the review more complicated as there are a lot of unnecessary changes. Can you please keep it in the original form?

Reply:
Could you please advise what the style guide for the repository is. I will then go reformat the files. Sorry about this.

Reply:
Oh no problem at all, actually is not a final style guide is defined. The issue is with changes in style the PR grows in size and makes it more difficult to review the important parts 😅. If we want to update the style guide we should create a separate PR only for that in the future.